src/main/java/com/yugabyte/sample/apps/AppConfig.java

```diff
@@ -165,4 +165,10 @@
 
   // The path to the certificate to be used for the SSL connection.
   public String sslCert = null;
+
+  public int num_devices = 100;
+
+  public int num_event_types = 100;
```

Comment:
Are these two used/needed? If yes add line-comments for them, otherwise remove.

Reply:
Yes added descriptions for these two as they are needed for Event data workloads